frontend/app/components/date-picker-field.tsx

```diff
@@ -311,7 +309,7 @@ interface DatePickerDayProps {
   id: string;
   label: string;
   name: string;
-  placeholder: string;
+  placeholder?: string;
```

Comment:
Remove `placeholder` prop since it's not used.

Reply:
it's not used in this component...doesn't mean it's never not going to be used though

Reply:
I'm not going to die on this hill, but I don't necessarily agree either.  Changes made.

Reply:
There's a blog posts and articles on that placeholder is bad for user's experience and I agree.

Reply:
https://www.nngroup.com/articles/form-design-placeholders/